python/README.md

```diff
@@ -1,8 +1,6 @@
-# Foxglove Studio WebSocket server
+# Foxglove WebSocket server
 
-This package provides a server implementation of the [Foxglove Studio WebSocket protocol](https://github.com/foxglove/ws-protocol), enabling [Foxglove Studio](https://github.com/foxglove/studio) to ingest arbitrary “live” streamed data.
-
-The protocol is encoding-agnostic, i.e. it can support Protobuf messages, JSON messages, etc. (as long as the desired encoding is supported by both client and server).
+This package provides an example server implementation of the [Foxglove WebSocket protocol](https://github.com/foxglove/ws-protocol), enabling [Foxglove Studio](https://github.com/foxglove/studio) to ingest arbitrary “live” streamed data.
```

Comment:
Not just an example server, this package provides a library meant to be used when writing a custom server

```suggestion
This package provides a server implementation of the [Foxglove WebSocket protocol](https://github.com/foxglove/ws-protocol), enabling [Foxglove Studio](https://github.com/foxglove/studio) to ingest arbitrary “live” streamed data.
```